client/src/components/Onboarding/OnboardingAdmin.jsx

```diff
@@ -237,17 +237,21 @@ const OnboardingAdmin = () => {
   const [showIncompleteOnly, setShowIncompleteOnly] = useState(false);
 
   const toggleShowIncomplete = () => {
+    dispatch({
+      type: 'FETCH_ONBOARDING_ADMIN_LIST',
+      payload: {
+        offset,
```

Comment:
This was intentional. I wanted to preserve the current position/ page that the user is on whenever the 'show incomplete' is toggled on or off. Let me know if that isn't the desired behavior.

Reply:
I have a use case: 
1. there are 200 users, and 20 users are incomplete. I think this number is more or less similar to prod scenario.
2. I'm on page 2 (incomplete user is check box is unchecked).
3. Toggle incomplete
4. Now the request go for offset and page matching page 2. But, 20 users does not fit into page 2. if the backend query is tolerant, then it will still return results when offset is beyond total results.

Try to see if you can repro this scenario. 

Toggling from incomplete on to off is generally ok because the count of total user is generally higher than incomplete.